### PR TITLE
Add a note section for unable to verify the first certificate in nodejs

### PR DESCRIPTION
**Pull request by Aaryan-R-S**
### Fixes Issue: [LIT-2509](https://linear.app/litprotocol/issue/LIT-2509/add-a-note-section-for-unable-to-verify-the-first-certificate-in)

# Description

Added a `> note:` section in the doc stating what to do if you’re facing `Error: unable to verify the first certificate in nodejs` 

Here - https://developer.litprotocol.com/v3/support/faq#11-is-web-assembly-supported-in-litactions 

Trying to update the docs for it!

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Introducing new feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: [description of change]
- [ ] Link 2: [...]

# Checklist:

General
- [ ] I have performed a self-review of my code
- [ ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ ] I have checked the additions are concise
- [ ] Language is consistent with existing documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ ] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

**Review by Ansonhkg — CHANGES_REQUESTED** [review]
Instead of having the same content in two different places, can we put that in [FAQ](https://developer.litprotocol.com/v3/support/faq/) and point user there?

**Comment by Aaryan-R-S** [comment]
Closing this PR as it was issue at my end only. Actually since I was on my campus WiFi, there are many firewalls so many domains were blocked. While minting pkp this is what was happening exaclty.

Please ensure that you don't have unnecessary firewalls preventing LIT calls to work properly. You can use VPNs or software like https://1.1.1.1/ to get access to more open and safer internet.

File: docs/support/faq.md [hunk 1]
```diff
@@ -118,6 +118,10 @@ Check out this PKP x WalletConnect example [here](https://github.com/LIT-Protoco
 
 Yes! Check out this [sample project.](https://github.com/dOrgJelli/lit-protocol-wasm-test/blob/d4b8873f9a5bceaf98e7f7a1bf325bf597cbfa40/src/App.js#L6-L51.)
 
+### 12. Why I'm getting following error message while making connection to Lit nodes "Error: Unable to verify the first certificate in nodejs"?
```
Comment (a1ttech) [inline docs/support/faq.md:121]: Minor comment on sentence structure. Consider:

"I am getting the following error when connecting to the Lit nodes: "Error: Unable to verify the first certificate in nodejs". What may be causing this?"

File: docs/support/faq.md [hunk 2]
```diff
@@ -118,6 +118,10 @@ Check out this PKP x WalletConnect example [here](https://github.com/LIT-Protoco
...
+
+If you're facing the `Error: unable to verify the first certificate` issue in Node.js while making connection to Lit nodes, then please ensure that you don't have unnecessary firewalls on your network preventing Lit calls to work properly. You can use VPNs or software like https://1.1.1.1/ to get access to more open and safer internet.
```
Comment (a1ttech) [inline docs/support/faq.md:123]: Minor comment on sentence structure. Consider:

"If you're facing this error while using Node.js, please first verify that you don't have unnecessary firewalls on your network that are preventing you from connecting to the Lit network properly. If you're using a VPN or software solution such as https://1.1.1.1/ , you should not face any issues"

**Comment by glitch003** [comment]
> Closing this PR as it was issue at my end only. Since I am on my campus WiFi, there are many firewalls so many domains are blocked. While minting PKPs this is what was happening exactly.
> 
> 
> 
> Please ensure that you don't have unnecessary firewalls preventing LIT calls to work properly. You can use VPNs or software like https://1.1.1.1/ to get access to more open and safer internet.

exactly which domain was blocked here?  we've never seen this issue before as far as i know so i hesitate to add it to the docs.

File: docs/support/faq.md (hunk 1 above)
Comment (Aaryan-R-S) [reply docs/support/faq.md:121]: Nicely done @a1ttech!

File: docs/support/faq.md (hunk 2 above)
Comment (Aaryan-R-S) [reply docs/support/faq.md:123]: Cool, done @a1ttech!

**Comment by Aaryan-R-S** [comment]
> > Closing this PR as it was issue at my end only. Since I am on my campus WiFi, there are many firewalls so many domains are blocked. While minting PKPs this is what was happening exactly.
> > Please ensure that you don't have unnecessary firewalls preventing LIT calls to work properly. You can use VPNs or software like https://1.1.1.1/ to get access to more open and safer internet.
> 
> exactly which domain was blocked here? we've never seen this issue before as far as i know so i hesitate to add it to the docs.

Hey @glitch003! I was talking about this API (and similar APIs) as shown in the attached SS:

image

**Comment by Aaryan-R-S** [comment]
@Ansonhkg can you review this PR?

**Comment by glitch003** [comment]
hey @Aaryan-R-S - i modified your changes a little just to make them clearer as to what's happening.  anyway it looks good now, merging.  thanks!

**Pull request merged**